src/index.html

```diff
@@ -21,7 +24,453 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        href="#"
+        class="img_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="img_header"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav__item">
```